packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -118,19 +101,30 @@ contract OPContractsManager is ISemver, Initializable {
         address permissionedDisputeGame2;
     }
 
+    /// @notice The latest implementation contracts for the OP Stack.
+    struct ImplementationContracts {
+        address l1ERC721BridgeImpl;
+        address optimismPortalImpl;
+        address systemConfigImpl;
+        address optimismMintableERC20FactoryImpl;
+        address l1CrossDomainMessengerImpl;
+        address l1StandardBridgeImpl;
+        address disputeGameFactoryImpl;
+        address delayedWETHImpl;
+        address mipsImpl;
+    }
+
     /// @notice Inputs required when initializing the OPContractsManager. To avoid 'StackTooDeep' errors,
     /// all necessary inputs (excluding immutables) for initialization are bundled together in this struct.
-    struct InitializerInputs {
+    struct InputContracts {
         Blueprints blueprints;
-        ImplementationSetter[] setters;
-        string release;
-        bool isLatest;
+        ImplementationContracts implementationContracts;
     }
```

Comment:
Is stack too deep still a concern? WOuld be nice if we can remove this nested `InputContracts` struct and flatten the constructor args into the individial `Blueprints` and `Implementations` structs

Reply:
Good call, the stack too deep error no longer exists. I changed this functionality here: https://github.com/ethereum-optimism/optimism/pull/12851/commits/c8068686416d6eda365a44d2bd74c9b33ac736d7#diff-32c676c7a2cc25dc934a436546eccc2c3484b44491cc1569dce382ffb13b84c7L198 

I still need to update the comment.